electron/main.ts

```diff
@@ -144,6 +155,14 @@ const openApplication = async (givenPath?: string) => {
   ElectronStore.initRenderer();
   const win = createWindow();
 
+  const missingDependecies = APP_DEPENDENCIES.filter(d => !which(d));
```

Comment:
move to common function that is called from here and above? - i.e. "checkMissingDependencies"